examples/Hello Wall/hello-wall.ifcx

```diff
@@ -126,13 +119,13 @@
     {
       "path": "7a187a90-3dcf-58cc-b3a6-51a9a407c55a",
       "attributes": {
-        "bsi::ifc::v5a::presentation::diffuseColor": [
+        "bsi::ifc::presentation::diffuseColor": [
```

Comment:
Why do we skip the version now? Is it explained somewhere what does that namespace `bsi::ifc` represent?